# English (Canada) translation for apport
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-11-19 08:43+0100\n"
"PO-Revision-Date: 2011-10-10 16:09+0000\n"
"Last-Translator: Itai Molenaar <Unknown>\n"
"Language-Team: English (Canada) <en_CA@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2014-04-04 04:38+0000\n"
"X-Generator: Launchpad (build 16976)\n"
"Language: \n"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:124
#: ../apport/ui.py:124
msgid "This package does not seem to be installed correctly"
msgstr "This package does not seem to be installed correctly"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:128
#: ../apport/ui.py:128
#, python-format
msgid ""
"This is not an official %s package. Please remove any third party package "
"and try again."
msgstr ""
"This is not an official %s package. Please remove any third party package "
"and try again."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:145
#: ../apport/ui.py:145
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:269
#: ../apport/ui.py:269
msgid "unknown program"
msgstr "unknown program"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:270
#: ../apport/ui.py:270
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Sorry, the program \"%s\" closed unexpectedly"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:272
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1306
#: ../apport/ui.py:272 ../apport/ui.py:1306
#, python-format
msgid "Problem in %s"
msgstr "Problem in %s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:273
#: ../apport/ui.py:273
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."

#. package does not exist
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:288
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:294
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:300
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:343
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:351
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:478
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:481
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:682
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1113
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1278
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1282
#: ../debian/tmp/usr/share/apport/testsuite/test_ui.py:276
#: ../debian/tmp/usr/share/apport/testsuite/test_ui.py:1049
#: ../apport/ui.py:288 ../apport/ui.py:294 ../apport/ui.py:300
#: ../apport/ui.py:343 ../apport/ui.py:351 ../apport/ui.py:478
#: ../apport/ui.py:481 ../apport/ui.py:682 ../apport/ui.py:1113
#: ../apport/ui.py:1278 ../apport/ui.py:1282
msgid "Invalid problem report"
msgstr "Invalid problem report"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:289
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1284
#: ../apport/ui.py:289 ../apport/ui.py:1284
msgid "This problem report is damaged and cannot be processed."
msgstr "This problem report is damaged and cannot be processed."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:295
#: ../apport/ui.py:295
msgid "The report belongs to a package that is not installed."
msgstr "The report belongs to a package that is not installed."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:301
#: ../apport/ui.py:301
msgid "An error occurred while attempting to process this problem report:"
msgstr "An error occurred while attempting to process this problem report:"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:344
#: ../apport/ui.py:344
msgid "You are not allowed to access this problem report."
msgstr "You are not allowed to access this problem report."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:347
#: ../apport/ui.py:347
msgid "Error"
msgstr "Error"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:348
#: ../apport/ui.py:348
msgid "There is not enough disk space available to process this report."
msgstr "There is not enough disk space available to process this report."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:432
#: ../apport/ui.py:432
msgid "No package specified"
msgstr "No package specified"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:433
#: ../apport/ui.py:433
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"You need to specify a package or a PID. See --help for more information."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:456
#: ../apport/ui.py:456
msgid "Permission denied"
msgstr "Permission denied"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:457
#: ../apport/ui.py:457
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:459
#: ../apport/ui.py:459
msgid "Invalid PID"
msgstr "Invalid PID"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:460
#: ../apport/ui.py:460
msgid "The specified process ID does not belong to a program."
msgstr "The specified process ID does not belong to a program."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:479
#: ../apport/ui.py:479
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Symptom script %s did not determine an affected package"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:482
#: ../apport/ui.py:482
#, python-format
msgid "Package %s does not exist"
msgstr "Package %s does not exist"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:506
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:694
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:699
#: ../apport/ui.py:506 ../apport/ui.py:694 ../apport/ui.py:699
msgid "Cannot create report"
msgstr "Cannot create report"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:521
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:567
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:584
#: ../apport/ui.py:521 ../apport/ui.py:567 ../apport/ui.py:584
msgid "Updating problem report"
msgstr "Updating problem report"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:522
#: ../apport/ui.py:522
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:531
#: ../apport/ui.py:531
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:568
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:585
#: ../apport/ui.py:568 ../apport/ui.py:585
msgid "No additional information collected."
msgstr "No additional information collected."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:636
#: ../apport/ui.py:636
msgid "What kind of problem do you want to report?"
msgstr "What kind of problem do you want to report?"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:653
#: ../apport/ui.py:653
msgid "Unknown symptom"
msgstr "Unknown symptom"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:654
#: ../apport/ui.py:654
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "The symptom \"%s\" is not known."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:685
#: ../apport/ui.py:685
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"After closing this message please click on an application window to report a "
"problem about it."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:695
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:700
#: ../apport/ui.py:695 ../apport/ui.py:700
msgid "xprop failed to determine process ID of the window"
msgstr "xprop failed to determine process ID of the window"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:714
#: ../apport/ui.py:714
msgid "%prog <report number>"
msgstr "%prog <report number>"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:716
#: ../apport/ui.py:716
msgid "Specify package name."
msgstr "Specify package name."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:718
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:769
#: ../apport/ui.py:718 ../apport/ui.py:769
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Add an extra tag to the report. Can be specified multiple times."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:748
#: ../apport/ui.py:748
msgid ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:751
#: ../apport/ui.py:751
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:753
#: ../apport/ui.py:753
msgid "Click a window as a target for filing a problem report."
msgstr "Click a window as a target for filing a problem report."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:755
#: ../apport/ui.py:755
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "Start in bug updating mode. Can take an optional --package."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:757
#: ../apport/ui.py:757
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:759
#: ../apport/ui.py:759
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:761
#: ../apport/ui.py:761
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:763
#: ../apport/ui.py:763
msgid "The provided pid is a hanging application."
msgstr "The provided pd is a hanging application."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:765
#: ../apport/ui.py:765
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:767
#: ../apport/ui.py:767
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:771
#: ../apport/ui.py:771
msgid "Print the Apport version number."
msgstr "Print the Apport version number."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:911
#: ../apport/ui.py:911
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"This will launch apport-retrace in a terminal window to examine the crash."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:912
#: ../apport/ui.py:912
msgid "Run gdb session"
msgstr "Run gdb session"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:913
#: ../apport/ui.py:913
msgid "Run gdb session without downloading debug symbols"
msgstr "Run gdb session without downloading debug symbols"

#. TRANSLATORS: %s contains the crash report file name
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:915
#: ../apport/ui.py:915
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Update %s with fully symbolic stack trace"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:991
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1001
#: ../apport/ui.py:991 ../apport/ui.py:1001
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"This problem report applies to a program which is not installed any more."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1016
#: ../apport/ui.py:1016
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"The problem happened with the program %s which changed since the crash "
"occurred."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1114
#: ../apport/ui.py:1114
msgid "Could not determine the package or source package name."
msgstr "Could not determine the package or source package name."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1132
#: ../apport/ui.py:1132
msgid "Unable to start web browser"
msgstr "Unable to start web browser"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1133
#: ../apport/ui.py:1133
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Unable to start web browser to open %s."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1232
#: ../apport/ui.py:1232
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr "Please enter your account information for the %s bug tracking system"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1244
#: ../apport/ui.py:1244
msgid "Network problem"
msgstr "Network problem"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1246
#: ../apport/ui.py:1246
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Cannot connect to crash database, please check your Internet connection."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1273
#: ../apport/ui.py:1273
msgid "Memory exhaustion"
msgstr "Memory exhaustion"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1274
#: ../apport/ui.py:1274
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Your system does not have enough memory to process this crash report."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1309
#: ../apport/ui.py:1309
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"The problem cannot be reported:\n"
"\n"
"%s"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1365
#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1372
#: ../apport/ui.py:1365 ../apport/ui.py:1372
msgid "Problem already known"
msgstr "Problem already known"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1366
#: ../apport/ui.py:1366
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:1373
#: ../apport/ui.py:1373
msgid "This problem was already reported to developers. Thank you!"
msgstr "This problem was already reported to developers. Thank you!"

#: ../debian/tmp/usr/bin/apport-cli.py:74 ../bin/apport-cli.py:74
msgid "Press any key to continue..."
msgstr "Press any key to continue..."

#: ../debian/tmp/usr/bin/apport-cli.py:81 ../bin/apport-cli.py:81
msgid "What would you like to do? Your options are:"
msgstr "What would you like to do? Your options are:"

#: ../debian/tmp/usr/bin/apport-cli.py:85 ../bin/apport-cli.py:85
#, python-format
msgid "Please choose (%s):"
msgstr "Please choose (%s):"

#: ../debian/tmp/usr/bin/apport-cli.py:148 ../bin/apport-cli.py:148
#, python-format
msgid "(%i bytes)"
msgstr "(%i bytes)"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:142 ../kde/apport-kde.py:342
#: ../debian/tmp/usr/share/apport/apport-kde.py:342
#: ../debian/tmp/usr/bin/apport-cli.py:150 ../gtk/apport-gtk.py:142
#: ../bin/apport-cli.py:150
msgid "(binary data)"
msgstr "(binary data)"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:177 ../kde/apport-kde.py:162
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:286
#: ../debian/tmp/usr/share/apport/apport-kde.py:162
#: ../debian/tmp/usr/bin/apport-cli.py:178 ../gtk/apport-gtk.py:177
#: ../bin/apport-cli.py:178
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:404
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:429
msgid "Send problem report to the developers?"
msgstr "Send problem report to the developers?"

#: ../debian/tmp/usr/bin/apport-cli.py:179 ../bin/apport-cli.py:179
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."

#: ../debian/tmp/usr/bin/apport-cli.py:182 ../bin/apport-cli.py:182
#, python-format
msgid "&Send report (%s)"
msgstr "&Send report (%s)"

#: ../debian/tmp/usr/bin/apport-cli.py:186 ../bin/apport-cli.py:186
msgid "&Examine locally"
msgstr "&Examine locally"

#: ../debian/tmp/usr/bin/apport-cli.py:190 ../bin/apport-cli.py:190
msgid "&View report"
msgstr "&View report"

#: ../debian/tmp/usr/bin/apport-cli.py:191 ../bin/apport-cli.py:191
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr "&Keep report file for sending later or copying to somewhere else"

#: ../debian/tmp/usr/bin/apport-cli.py:192 ../bin/apport-cli.py:192
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Cancel and &ignore future crashes of this program version"

#: ../debian/tmp/usr/bin/apport-cli.py:194
#: ../debian/tmp/usr/bin/apport-cli.py:271
#: ../debian/tmp/usr/bin/apport-cli.py:303
#: ../debian/tmp/usr/bin/apport-cli.py:324 ../bin/apport-cli.py:194
#: ../bin/apport-cli.py:271 ../bin/apport-cli.py:303 ../bin/apport-cli.py:324
msgid "&Cancel"
msgstr "&Cancel"

#: ../debian/tmp/usr/bin/apport-cli.py:222 ../bin/apport-cli.py:222
msgid "Problem report file:"
msgstr "Problem report file:"

#: ../debian/tmp/usr/bin/apport-cli.py:228
#: ../debian/tmp/usr/bin/apport-cli.py:233 ../bin/apport-cli.py:228
#: ../bin/apport-cli.py:233
msgid "&Confirm"
msgstr "&Confirm"

#: ../debian/tmp/usr/bin/apport-cli.py:232 ../bin/apport-cli.py:232
#, python-format
msgid "Error: %s"
msgstr "Error: %s"

#: ../kde/apport-kde.py:387 ../debian/tmp/usr/share/apport/apport-kde.py:387
#: ../debian/tmp/usr/bin/apport-cli.py:238 ../bin/apport-cli.py:238
msgid "Collecting problem information"
msgstr "Collecting problem information"

#: ../debian/tmp/usr/bin/apport-cli.py:239 ../bin/apport-cli.py:239
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."

#: ../gtk/apport-gtk.ui.h:13 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:13
#: ../kde/apport-kde.py:415 ../debian/tmp/usr/share/apport/apport-kde.py:415
#: ../debian/tmp/usr/bin/apport-cli.py:251 ../bin/apport-cli.py:251
msgid "Uploading problem information"
msgstr "Uploading problem information"

#: ../debian/tmp/usr/bin/apport-cli.py:252 ../bin/apport-cli.py:252
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."

#: ../debian/tmp/usr/bin/apport-cli.py:302 ../bin/apport-cli.py:302
msgid "&Done"
msgstr "&Done"

#: ../debian/tmp/usr/bin/apport-cli.py:308 ../bin/apport-cli.py:308
msgid "none"
msgstr "none"

#: ../debian/tmp/usr/bin/apport-cli.py:309 ../bin/apport-cli.py:309
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Selected: %s. Multiple choices:"

#: ../debian/tmp/usr/bin/apport-cli.py:325 ../bin/apport-cli.py:325
msgid "Choices:"
msgstr "Choices:"

#: ../debian/tmp/usr/bin/apport-cli.py:339 ../bin/apport-cli.py:339
msgid "Path to file (Enter to cancel):"
msgstr "Path to file (Enter to cancel):"

#: ../debian/tmp/usr/bin/apport-cli.py:345 ../bin/apport-cli.py:345
msgid "File does not exist."
msgstr "File does not exist."

#: ../debian/tmp/usr/bin/apport-cli.py:347 ../bin/apport-cli.py:347
msgid "This is a directory."
msgstr "This is a directory."

#: ../debian/tmp/usr/bin/apport-cli.py:353 ../bin/apport-cli.py:353
msgid "To continue, you must visit the following URL:"
msgstr "To continue, you must visit the following URL:"

#: ../debian/tmp/usr/bin/apport-cli.py:355 ../bin/apport-cli.py:355
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."

#: ../debian/tmp/usr/bin/apport-cli.py:357 ../bin/apport-cli.py:357
msgid "Launch a browser now"
msgstr "Launch a browser now"

#: ../debian/tmp/usr/bin/apport-cli.py:371 ../bin/apport-cli.py:371
msgid "No pending crash reports. Try --help for more information."
msgstr "No pending crash reports. Try --help for more information."

#: ../data/apportcheckresume.py:68
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:68
msgid ""
"This occured during a previous suspend and prevented it from resuming "
"properly."
msgstr ""
"This occurred during a previous suspend and prevented it from resuming "
"properly."

#: ../data/apportcheckresume.py:70
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:70
msgid ""
"This occured during a previous hibernate and prevented it from resuming "
"properly."
msgstr ""
"This occurred during a previous hibernate and prevented it from resuming "
"properly."

#: ../data/apportcheckresume.py:75
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:75
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."

#: ../debian/tmp/usr/bin/apport-retrace.py:31 ../bin/apport-retrace.py:31
msgid "%prog [options] <apport problem report | crash ID>"
msgstr "%prog [options] <apport problem report | crash ID>"

#: ../debian/tmp/usr/bin/apport-retrace.py:33 ../bin/apport-retrace.py:33
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Do not put the new traces into the report, but write them to stdout."

#: ../debian/tmp/usr/bin/apport-retrace.py:35 ../bin/apport-retrace.py:35
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"

#: ../debian/tmp/usr/bin/apport-retrace.py:37 ../bin/apport-retrace.py:37
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Write modified report to given file instead of changing the original report"

#: ../debian/tmp/usr/bin/apport-retrace.py:39 ../bin/apport-retrace.py:39
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr "Remove the core dump from the report after stack trace regeneration"

#: ../debian/tmp/usr/bin/apport-retrace.py:41 ../bin/apport-retrace.py:41
msgid "Override report's CoreFile"
msgstr "Override report's CoreFile"

#: ../debian/tmp/usr/bin/apport-retrace.py:43 ../bin/apport-retrace.py:43
msgid "Override report's ExecutablePath"
msgstr "Override report's ExecutablePath"

#: ../debian/tmp/usr/bin/apport-retrace.py:45 ../bin/apport-retrace.py:45
msgid "Override report's ProcMaps"
msgstr "Override report's ProcMaps"

#: ../debian/tmp/usr/bin/apport-retrace.py:47 ../bin/apport-retrace.py:47
msgid "Rebuild report's Package information"
msgstr "Rebuild report's Package information"

#: ../debian/tmp/usr/bin/apport-retrace.py:49 ../bin/apport-retrace.py:49
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."

#: ../debian/tmp/usr/bin/apport-retrace.py:51 ../bin/apport-retrace.py:51
msgid ""
"Report download/install progress when installing packages into sandbox"
msgstr ""
"Report download/install progress when installing packages into sandbox"

#: ../debian/tmp/usr/bin/apport-retrace.py:53 ../bin/apport-retrace.py:53
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Prepend timestamps to log messages, for batch operation"

#: ../debian/tmp/usr/bin/apport-retrace.py:55 ../bin/apport-retrace.py:55
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Cache directory for packages downloaded in the sandbox"

#: ../debian/tmp/usr/bin/apport-retrace.py:57 ../bin/apport-retrace.py:57
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."

#: ../bin/apport-valgrind.py:66 ../debian/tmp/usr/bin/apport-retrace.py:59
#: ../bin/apport-retrace.py:59 ../debian/tmp/usr/bin/apport-valgrind.py:66
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"Install an extra package into the sandbox (can be specified multiple times)"

#: ../debian/tmp/usr/bin/apport-retrace.py:61 ../bin/apport-retrace.py:61
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"

#: ../debian/tmp/usr/bin/apport-retrace.py:63 ../bin/apport-retrace.py:63
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."

#: ../debian/tmp/usr/bin/apport-retrace.py:65 ../bin/apport-retrace.py:65
msgid ""
"Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Path to the duplicate sqlite database (default: no duplicate checking)"

#: ../debian/tmp/usr/bin/apport-retrace.py:70 ../bin/apport-retrace.py:70
msgid "incorrect number of arguments; use --help for a short help"
msgstr "incorrect number of arguments; use --help for a short help"

#: ../debian/tmp/usr/bin/apport-retrace.py:76 ../bin/apport-retrace.py:76
msgid "You cannot use -C without -S. Stopping."
msgstr "You cannot use -C without -S. Stopping."

#: ../debian/tmp/usr/bin/apport-retrace.py:80 ../bin/apport-retrace.py:80
msgid ""
"you either need to do a local operation (-s, -g, -o) or supply an "
"authentication file (--auth); see --help for a short help"
msgstr ""
"you either need to do a local operation (-s, -g, -o) or supply an "
"authentication file (--auth); see --help for a short help"

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../debian/tmp/usr/bin/apport-retrace.py:114 ../bin/apport-retrace.py:114
msgid "OK to send these as attachments? [y/n]"
msgstr "OK to send these as attachments? [y/n]"

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde.desktop.in.h:1
#: ../kde/apport-kde-mime.desktop.in.h:1
msgid "Report a problem..."
msgstr "Report a problem..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde.desktop.in.h:2
#: ../kde/apport-kde-mime.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Report a malfunction to the developers"

#: ../data/kernel_oops.py:30 ../debian/tmp/usr/share/apport/kernel_oops.py:30
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Your system might become unstable now and might need to be restarted."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:157 ../gtk/apport-gtk.py:157
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Sorry, the application %s has stopped unexpectedly."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:160 ../gtk/apport-gtk.py:160
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Sorry, %s has closed unexpectedly."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:165 ../kde/apport-kde.py:176
#: ../kde/apport-kde.py:217
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:110
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:254
#: ../debian/tmp/usr/share/apport/apport-kde.py:176
#: ../debian/tmp/usr/share/apport/apport-kde.py:217 ../gtk/apport-gtk.py:165
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:108
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:300
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Sorry, %s has experienced an internal error."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:186 ../kde/apport-kde.py:170
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:292
#: ../debian/tmp/usr/share/apport/apport-kde.py:170 ../gtk/apport-gtk.py:186
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:412
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:437
msgid "Send"
msgstr "Send"

#: ../gtk/apport-gtk.ui.h:7 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:7
#: ../debian/tmp/usr/share/apport/apport-gtk.py:204
#: ../debian/tmp/usr/share/apport/apport-gtk.py:573 ../kde/apport-kde.py:271
#: ../debian/tmp/usr/share/apport/apport-kde.py:271 ../gtk/apport-gtk.py:204
#: ../gtk/apport-gtk.py:573
msgid "Show Details"
msgstr "Show Details"

#: ../gtk/apport-gtk.ui.h:10 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:10
#: ../debian/tmp/usr/share/apport/apport-gtk.py:216
#: ../debian/tmp/usr/share/apport/apport-gtk.py:284
#: ../debian/tmp/usr/share/apport/apport-gtk.py:303 ../kde/apport-kde.py:210
#: ../kde/apport-kde.py:223
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:115
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:141
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:153
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:186
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:262
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:332
#: ../debian/tmp/usr/share/apport/apport-kde.py:210
#: ../debian/tmp/usr/share/apport/apport-kde.py:223 ../gtk/apport-gtk.py:216
#: ../gtk/apport-gtk.py:284 ../gtk/apport-gtk.py:303
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:115
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:144
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:157
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:192
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:310
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:350
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:479
msgid "Continue"
msgstr "Continue"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:228 ../gtk/apport-gtk.py:228
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:276
msgid "Force Closed"
msgstr "Force Closed"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:229
#: ../debian/tmp/usr/share/apport/apport-gtk.py:288 ../kde/apport-kde.py:214
#: ../kde/apport-kde.py:361
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:226
#: ../debian/tmp/usr/share/apport/apport-kde.py:214
#: ../debian/tmp/usr/share/apport/apport-kde.py:361 ../gtk/apport-gtk.py:229
#: ../gtk/apport-gtk.py:288
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:232
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:273
msgid "Relaunch"
msgstr "Relaunch"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:238 ../gtk/apport-gtk.py:238
#, python-format
msgid "The application %s has stopped responding."
msgstr "The application %s has stopped responding."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:242 ../gtk/apport-gtk.py:242
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "The program \"%s\" has stopped responding."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:257 ../kde/apport-kde.py:184
#: ../debian/tmp/usr/share/apport/apport-kde.py:184 ../gtk/apport-gtk.py:257
#, python-format
msgid "Package: %s"
msgstr "Package: %s"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:264 ../kde/apport-kde.py:190
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:136
#: ../debian/tmp/usr/share/apport/apport-kde.py:190 ../gtk/apport-gtk.py:264
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:137
msgid "Sorry, a problem occurred while installing software."
msgstr "Sorry, a problem occurred while installing software."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:273
#: ../debian/tmp/usr/share/apport/apport-gtk.py:292 ../kde/apport-kde.py:197
#: ../debian/tmp/usr/share/apport/apport-kde.py:197 ../gtk/apport-gtk.py:273
#: ../gtk/apport-gtk.py:292
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "The application %s has experienced an internal error."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:276 ../kde/apport-kde.py:201
#: ../debian/tmp/usr/share/apport/apport-kde.py:201 ../gtk/apport-gtk.py:276
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "The application %s has closed unexpectedly."

#: ../gtk/apport-gtk.ui.h:9 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:9
#: ../debian/tmp/usr/share/apport/apport-gtk.py:287 ../kde/apport-kde.py:213
#: ../debian/tmp/usr/share/apport/apport-kde.py:213 ../gtk/apport-gtk.py:287
msgid "Leave Closed"
msgstr "Leave Closed"

#: ../gtk/apport-gtk.ui.h:4 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:4
#: ../debian/tmp/usr/share/apport/apport-gtk.py:300 ../kde/apport-kde.py:220
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_kde.py:256
#: ../debian/tmp/usr/share/apport/apport-kde.py:220 ../gtk/apport-gtk.py:300
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:302
#: ../debian/tmp/usr/share/apport/testsuite/test_ui_gtk.py:342
msgid "If you notice further problems, try restarting the computer."
msgstr "If you notice further problems, try restarting the computer."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:304 ../kde/apport-kde.py:224
#: ../debian/tmp/usr/share/apport/apport-kde.py:224 ../gtk/apport-gtk.py:304
msgid "Ignore future problems of this type"
msgstr "Ignore future problems of this type"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:577 ../kde/apport-kde.py:268
#: ../debian/tmp/usr/share/apport/apport-kde.py:268 ../gtk/apport-gtk.py:577
msgid "Hide Details"
msgstr "Hide Details"

#: ../bin/apport-valgrind.py:37 ../debian/tmp/usr/bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "See man page for details."

#: ../bin/apport-valgrind.py:43 ../debian/tmp/usr/bin/apport-valgrind.py:43
msgid "specify the log file name produced by valgrind"
msgstr "specify the log file name produced by valgrind"

#: ../bin/apport-valgrind.py:46 ../debian/tmp/usr/bin/apport-valgrind.py:46
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"

#: ../bin/apport-valgrind.py:50 ../debian/tmp/usr/bin/apport-valgrind.py:50
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."

#: ../bin/apport-valgrind.py:54 ../debian/tmp/usr/bin/apport-valgrind.py:54
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"

#: ../bin/apport-valgrind.py:58 ../debian/tmp/usr/bin/apport-valgrind.py:58
msgid ""
"report download/install progress when installing packages into sandbox"
msgstr ""
"report download/install progress when installing packages into sandbox"

#: ../bin/apport-valgrind.py:62 ../debian/tmp/usr/bin/apport-valgrind.py:62
msgid ""
"the executable that is run under valgrind's memcheck tool  for memory leak "
"detection"
msgstr ""
"the executable that is run under valgrind's memcheck tool  for memory leak "
"detection"

#: ../bin/apport-valgrind.py:97 ../debian/tmp/usr/bin/apport-valgrind.py:97
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Error: %s is not an executable. Stopping."

#: ../kde/apport-kde.py:294 ../debian/tmp/usr/share/apport/apport-kde.py:294
msgid "Username:"
msgstr "Username:"

#: ../kde/apport-kde.py:295 ../debian/tmp/usr/share/apport/apport-kde.py:295
msgid "Password:"
msgstr "Password:"

#: ../kde/apport-kde.py:386 ../debian/tmp/usr/share/apport/apport-kde.py:386
msgid "Collecting Problem Information"
msgstr "Collecting Problem Information"

#: ../kde/apport-kde.py:388 ../debian/tmp/usr/share/apport/apport-kde.py:388
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."

#: ../kde/apport-kde.py:414 ../debian/tmp/usr/share/apport/apport-kde.py:414
msgid "Uploading Problem Information"
msgstr "Uploading Problem Information"

#: ../gtk/apport-gtk.ui.h:15 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:15
#: ../kde/apport-kde.py:416 ../debian/tmp/usr/share/apport/apport-kde.py:416
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."

#: ../gtk/apport-gtk.ui.h:1 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:1
#: ../kde/apport-kde.py:450 ../kde/apport-kde.py:486
#: ../debian/tmp/usr/share/apport/apport-kde.py:450
#: ../debian/tmp/usr/share/apport/apport-kde.py:486
msgid "Apport"
msgstr "Apport"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Collect system information"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Authentication is required to collect system information for this problem "
"report"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "System problem reports"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid ""
"Please enter your password to access problem reports of system programs"
msgstr ""
"Please enter your password to access problem reports of system programs"

#: ../gtk/apport-gtk.ui.h:2 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:2
msgid "Crash report"
msgstr "Crash report"

#: ../gtk/apport-gtk.ui.h:3 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:3
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Sorry, an internal error happened.</b></big>"

#: ../gtk/apport-gtk.ui.h:5 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:5
msgid "Send an error report to help fix this problem"
msgstr "Send an error report to help fix this problem"

#: ../gtk/apport-gtk.ui.h:6 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:6
msgid "Ignore future problems of this program version"
msgstr "Ignore future problems of this program version"

#: ../gtk/apport-gtk.ui.h:8 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:8
msgid "_Examine locally"
msgstr "_Examine locally"

#: ../gtk/apport-gtk.ui.h:11 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:11
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Collecting problem information</b></big>"

#: ../gtk/apport-gtk.ui.h:12 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:12
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"Information is being collected that may help the developers fix the problem "
"you report."

#: ../gtk/apport-gtk.ui.h:14 ../debian/tmp/usr/share/apport/apport-gtk.ui.h:14
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Uploading problem information</b></big>"

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport crash file"

#: ../debian/tmp/usr/bin/apport-unpack.py:22 ../bin/apport-unpack.py:22
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr "Usage: %s <report> <target directory>"

#: ../debian/tmp/usr/bin/apport-unpack.py:46 ../bin/apport-unpack.py:46
msgid "Destination directory exists and is not empty."
msgstr "Destination directory exists and is not empty."

#~ msgid "Your system encountered a serious kernel problem."
#~ msgstr "Your system encountered a serious kernel problem."

#, python-format
#~ msgid "Sorry, %s closed unexpectedly"
#~ msgstr "Sorry, %s closed unexpectedly"

#~ msgid "You can help the developers to fix the problem by reporting it."
#~ msgstr "You can help the developers to fix the problem by reporting it."

#~ msgid ""
#~ "<big><b>Send problem report to the developers?</b></big>\n"
#~ "\n"
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."
#~ msgstr ""
#~ "<big><b>Send problem report to the developers?</b></big>\n"
#~ "\n"
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."

#~ msgid "Application problem"
#~ msgstr "Application problem"

#, no-c-format
#~ msgid ""
#~ "<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
#~ "install or upgrade.</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
#~ "install or upgrade.</span>"

#, no-c-format, python-format
#~ msgid "Reduced report (slow Internet connection; %s)"
#~ msgstr "Reduced report (slow Internet connection; %s)"

#~ msgid "Restart _Program"
#~ msgstr "Restart _Program"

#~ msgid ""
#~ "This will remove some large items from the report. These are very useful for "
#~ "developers to debug the problem, but might be too big for you to upload if "
#~ "you have a slow internet connection."
#~ msgstr ""
#~ "This will remove some large items from the report. These are very useful for "
#~ "developers to debug the problem, but might be too big for you to upload if "
#~ "you have a slow Internet connection."

#~ msgid "Package problem"
#~ msgstr "Package problem"

#~ msgid "Kernel problem"
#~ msgstr "Kernel problem"

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other "
#~ "private information), you can help to improve the application by reporting "
#~ "the problem."
#~ msgstr ""
#~ "If you were not doing anything confidential (entering passwords or other "
#~ "private information), you can help to improve the application by reporting "
#~ "the problem."

#~ msgid "Content of the report"
#~ msgstr "Content of the report"

#, no-c-format, python-format
#~ msgid "Complete report (recommended; %s)"
#~ msgstr "Complete report (recommended; %s)"

#~ msgid ""
#~ "You can help the developers to fix the package by reporting the problem."
#~ msgstr ""
#~ "You can help the developers to fix the package by reporting the problem."

#~ msgid "_Ignore future crashes of this program version"
#~ msgstr "_Ignore future crashes of this program version"

#~ msgid "_Report Problem..."
#~ msgstr "_Report Problem..."

#~ msgid "_Send Report"
#~ msgstr "_Send Report"

#, python-format
#~ msgid "Sorry, the program \"%s\" closed unexpectedly."
#~ msgstr "Sorry, the program \"%s\" closed unexpectedly."

#, python-format
#~ msgid "Sorry, %s closed unexpectedly."
#~ msgstr "Sorry, %s closed unexpectedly."

#~ msgid "&Details..."
#~ msgstr "&Details..."

#~ msgid "&Send"
#~ msgstr "&Send"

#~ msgid ""
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."
#~ msgstr ""
#~ "After the problem report has been sent, please fill out the form in the "
#~ "automatically opened web browser."

#~ msgid "Generic error"
#~ msgstr "Generic error"

#~ msgid "&Ignore future crashes of this program version"
#~ msgstr "&Ignore future crashes of this program version"

#~ msgid "Restart &Program"
#~ msgstr "Restart &Program"

#~ msgid "&Report Problem..."
#~ msgstr "&Report Problem..."

#, python-format
#~ msgid "Sorry, the package \"%s\" failed to install or upgrade."
#~ msgstr "Sorry, the package \"%s\" failed to install or upgrade."

#, python-format
#~ msgid "The package \"%s\" failed to install or upgrade."
#~ msgstr "The package \"%s\" failed to install or upgrade."

#~ msgid ""
#~ "If you were not doing anything confidential (entering passwords or other\n"
#~ "private information), you can help to improve the application by reporting\n"
#~ "the problem."
#~ msgstr ""
#~ "If you were not doing anything confidential (entering passwords or other\n"
#~ "private information), you can help to improve the application by reporting\n"
#~ "the problem."

#, python-format
#~ msgid "%s closed unexpectedly on %s at %s."
#~ msgstr "%s closed unexpectedly on %s at %s."

#, python-format
#~ msgid "&Send complete report (recommended; %s)"
#~ msgstr "&Send complete report (recommended; %s)"

#, python-format
#~ msgid "Send &reduced report (slow Internet connection; %s)"
#~ msgstr "Send &reduced report (slow Internet connection; %s)"

#, python-format
#~ msgid "This is not a genuine %s package"
#~ msgstr "This is not a genuine %s package"

#~ msgid "Send this data to the developers?"
#~ msgstr "Send this data to the developers?"
